app/src/main/java/com/yacgroup/yacguide/AboutActivity.kt

```diff
@@ -36,12 +38,33 @@ class AboutActivity : BaseNavigationActivity() {
         super.onCreate(savedInstanceState)
         this.title = getString(R.string.menu_about, getString(R.string.app_name))
 
-        val textViewAppVersion: TextView = findViewById(R.id.textViewAppVersion)
-        textViewAppVersion.text = "Version ${getAppVersion()}"
-        textViewAppVersion.gravity = Gravity.CENTER_HORIZONTAL
+        _createEntry(getString(R.string.data_src), getString(R.string.data_src_url),null)
+        _createEntry(getString(R.string.privacy_policy), null) { _showPrivacyPolicy() }
+        _createEntry(
+                getString(R.string.source_code_and_support),
+                getString(R.string.github_url),
+                null )
+        _createEntry(getString(R.string.license), getString(R.string.license_url),null)
+        _createEntry(getString(R.string.version), _getAppVersion(), null)
     }
 
-    private fun getAppVersion() : String {
+    private fun _createEntry(
+            title: String,
+            description: String?,
+            callback: ((view: View) -> Unit)?) {
```

Comment:
Why does the callback expect a view as parameter?

Reply:
Because `setOnClickListener` expects this. I fixed this.